[2025-03-19T14:21:16.628+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-19T14:21:16.647+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: enrich_content_dag.update_postgres manual__2025-03-19T14:20:49.502728+00:00 [queued]>
[2025-03-19T14:21:16.656+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: enrich_content_dag.update_postgres manual__2025-03-19T14:20:49.502728+00:00 [queued]>
[2025-03-19T14:21:16.657+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 5
[2025-03-19T14:21:16.669+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): update_postgres> on 2025-03-19 14:20:49.502728+00:00
[2025-03-19T14:21:16.676+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'enrich_content_dag', 'update_postgres', 'manual__2025-03-19T14:20:49.502728+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/Enrich_content.py', '--cfg-path', '/tmp/tmpnj372szw']
[2025-03-19T14:21:16.678+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4262) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-19T14:21:16.679+0000] {standard_task_runner.py:105} INFO - Job 37: Subtask update_postgres
[2025-03-19T14:21:16.679+0000] {standard_task_runner.py:72} INFO - Started process 4264 to run task
[2025-03-19T14:21:16.717+0000] {task_command.py:467} INFO - Running <TaskInstance: enrich_content_dag.update_postgres manual__2025-03-19T14:20:49.502728+00:00 [running]> on host f8578d95b82a
[2025-03-19T14:21:16.772+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='enrich_content_dag' AIRFLOW_CTX_TASK_ID='update_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-03-19T14:20:49.502728+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-19T14:20:49.502728+00:00'
[2025-03-19T14:21:16.773+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-19T14:21:16.832+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-19T14:21:16.840+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-19T14:21:16.841+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=enrich_content_dag, task_id=update_postgres, run_id=manual__2025-03-19T14:20:49.502728+00:00, execution_date=20250319T142049, start_date=20250319T142116, end_date=20250319T142116
[2025-03-19T14:21:16.894+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-19T14:21:16.907+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-19T14:21:16.909+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
